hera_cal/reflections.py

```diff
@@ -579,17 +582,24 @@ def sv_decomp(self, dfft, wgts=None, flags=None, keys=None, Nkeep=None,
 
     def project_svd_modes(self, dfft, umodes=None, svals=None, vmodes=None):
         """
-        Given two of the 3 SVD output vectors, project the input dfft data onto
-        the last remaining SVD vector.
+        Given two of the 3 SVD output matrices, project them onto the input dfft data
+        to estimate the last remaining SVD matrix. Note U and V are unitary matrices.
+
+        If
+            D = U S V
+        then
+            U = D V_dagger S_inv
+            S = U_dagger D V_dagger
+            V = S_inv U_dagger D
 
         Args:
             dfft : DataContainer, holds time-delay waterfall visibilities
-            umodes : DataContainer, holds SVD umodes
-            svals : DataContainer, holds SVD singular values
-            vmodes : DataContainer, holds SVD vmodes
+            umodes : DataContainer, holds SVD umodes, shape (Ntimes, Nmodes)
+            svals : DataContainer, holds SVD singular values, shape (Nmodes,)
```

Comment:
Clarify the assumption that nFreqs > nModes, which makes this shape correct.

Reply:
I don't think that is required actually